iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergMetaHook.java

```diff
@@ -1181,6 +1180,35 @@ public void preDropPartitions(org.apache.hadoop.hive.metastore.api.Table hmsTabl
     context.putToProperties(HiveMetaStoreClient.SKIP_DROP_PARTITION, "true");
   }
 
+  private static void validatePartitionSpec(SearchArgument sarg, Table icebergTbl) {
```

Comment:
could we change to `PartitionSpec partitionSpec`->  icebergTbl.spec()?

Reply:
done